packages/mdc-select/foundation.ts

```diff
@@ -227,9 +230,8 @@ export class MDCSelectFoundation extends MDCFoundation<MDCSelectAdapter> {
   /**
    * Handles value changes, via change event or programmatic updates.
    */
-  handleChange() {
+  handleChange(notifyChange = true) {
```

Comment:
We can remove boolean from here as well and make handleChange always notify the change. 

We'll need to abstract out the code from `handleChange` to private method so it can be reused in other places (`setValue`, `setSelectedIndex`, etc). It is a good practice to bind `handle*` methods to only event callbacks in component implementation so we won't break other calls when we introduce any changes to `handleChange` callback in future. 

Currently `handleChange` is called in component implementation and also used has helper function within foundation.

Reply:
Updated. `setValue` and `setSelectedIndex` now both won't fire a change event since they are both programmatic changes.